Description: Work around gpg-error-config removal in new version of libgpg-error-dev
 /usr/bin/gpg-error-config is removed in libgpg-error-dev 1.46-1. Use pkgconf
 to work around this. Version 0.6 of this crate removed this usage.
Last-Update: 2022-11-15
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/build.rs
+++ b/build.rs
@@ -87,7 +87,7 @@
 }
 
 fn try_config<S: Into<OsString>>(proj: &Project, path: S) -> Result<Config> {
-    let path = path.into();
+    let path: OsString = "pkgconf gpg-error".into();
     let mut cmd = path.clone();
     cmd.push(" --version");
     let version = output(Command::new("sh").arg("-c").arg(cmd))?;
